mappings/net/minecraft/class_5284.mapping

```diff
@@ -1,3 +1,15 @@
 CLASS net/minecraft/class_5284
```

Comment:
seems kinda weird how this class has all these names but still isn't named itself, but I guess it's ok

Reply:
Has been mapped in #1353

Reply:
yeah i dropped the name because theirs was better